data_generators/generators.py

```diff
@@ -92,7 +93,7 @@ def __init__(self, mentee_id, mentor_id):
         self.ticket_id = generate_uuid(16)
         self.mentee_id = mentee_id
         self.mentor_id = mentor_id
-        self.text = choice(self.feedback["Review"])[:800]
+        self.text = sentiment_rank(choice(self.feedback["Review"])[:800])
```

Comment:
Almost, but not quite what we need. This would remove the review itself and place the sentiment_rank in the text field. We should instead, make a new field to store the sentiment_rank.